courses/migrations/0006_author_name_optional.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
```

Comment:
We shouldn't ever be editing migrations that have already been committed like these. Can you change this to keep `0006_merge` and move the `course` `AlterField` change into a separate migration? This isn't terribly important for right now since there's no data but at the very least it gives @bdero fewer things to worry about on deploy, since we already have an instance which is applying migrations.